frontend/public/components/factory/Table/active-columns-hook.ts

```diff
@@ -37,10 +37,6 @@ export const useActiveColumns = <D = any>({
             }),
           );
 
-    if (showNamespaceOverride && !activeColumnIDs.has('namespace')) {
```

Comment:
Will this show the namespace in other tables where we want to hide it?

Reply:
@spadgett I actually think this would have the opposite effect. It looks like certain pages use "showNamespaceOverride" to ensure that the namespace column is always shown, regardless of user settings. Removing this would mean that we respect the user setting instead. IMO, it makes sense to respect the user setting always. If we want to do this though, we also need to remove the arg from the hook and remove all references to this override in components that currently use it.